site/public/css/server.css

```diff
@@ -501,6 +501,7 @@ td > a.active-sort {
 
 .option {
     margin-bottom: 20px;
+    margin-left: 40px;
```

Comment:
This CSS class is used all over the codebase.  Adding a margin here is risky because it has the potential to inadvertently change the styling elsewhere.

This change should be applied to a more specific identifier, if it is needed.